machines/flush-cache.js

```diff
@@ -42,16 +42,6 @@ module.exports = {
       }
     },
 
-    failed: {
```

Comment:
@luislobo Hard to say here-- on one hand, it makes sense to keep this in case other implementations work differently (I'm not sure about https://github.com/memcached/memcached/wiki/Commands#flush_all).  But I get what you're thinking-- seems unlikely that there'd be issues that aren't related to a bad connection...  Your call.